tempesta_fw/http.c

```diff
@@ -1998,6 +2182,19 @@ tfw_http_req_cache_cb(TfwHttpReq *req, TfwHttpResp *resp)
 	tfw_http_req_zap_error(&equeue);
 	goto conn_put;
 
+send_503:
+	/*
+	 * Requested resource can't be challenged. Don't break response-request
+	 * queue on client side by dropping the request.
+	 */
+	HTTP_SEND_RESP(req, 503, "request dropped: can't send JS challenge.");
+	TFW_INC_STAT_BH(clnt.msgs_otherr);
```

Comment:
This and all the below are security events, so it seem `msgs_filtout` counter must be incremented instead.